settings/moonchild-signalk-ws-settings.json

```diff
@@ -0,0 +1,48 @@
+{
+        "vessel": {
+                "name"  : "Moonchild",
+                "brand" : "Catalina",
+                "type"  : "320",
+                "uuid"  : "self",
+
+                "dimensions": {
+                        "length": 9.9,
+                        "width": 3.6,
+                        "mast": 14.5,
+                        "depthTransducer": 0.6,
+                        "keel": 1.8
+                }
+        },
+    "mdns": true,
+	"interfaces": {
+    "rest": false,
+    "ws": true,
+    "bower": false,
```

Comment:
That is the configuration file I use for my boat and like others, it has boat specific details and settings. I am not married  to it, will push a change. That said, it is a bit mixed here and there and not clear or inconsistent between a) what is part of the spec b) what is part of the implementation c) what is unique to a specific configuration d) what is personal choice. That makes it harder to onboard yourself.